draft-ietf-ppm-dap.md

```diff
@@ -1080,6 +1076,7 @@ struct {
 struct {
   TaskID task_id;
   AggregationJobID job_id;
+  Extension extensions<0..2^16-1>;
```

Comment:
I think you should explain how this relates to the `extensions` field in `CollectReq`. In particular, I think the only reason that a `CollectReq` would ever contain extensions is so that they can be relayed to the helper. If the collector wanted to send something special to the leader, it could send an extra header in the HTTP request that sends the `CollectReq`. A leader that wants to add something non-standard to an `AggregateInitializeReq` could do much the same thing. It's only the collector<->helper case that needs some kind of in-band extensions scheme, since they never communicate directly (yet).

Reply:
Is this `extensions` meant to replace the `ReportShare.encrypted_input_share .payload.extensions` eventually? If not, what's the relation between these two extensions, can people introduce an extension only relevant to `AggregateInitializeReq` but not clients?